GitTfs/Core/GitTfsRemote.cs

```diff
@@ -485,8 +485,10 @@ private string ProcessChangeset(ITfsChangeset changeset, LogEntry log)
                 if (changeset.Summary.Workitems.Any())
                 {
                     var workItemIds = TranslateWorkItems(changeset.Summary.Workitems.Select(wi => wi.Id.ToString()));
+                    var workItemTitles = changeset.Summary.Workitems.Select(wi => wi.Title);
                     if (workItemIds != null)
                     {
+                        log.Log += "\n" + string.Join(", ", workItemTitles.Select(s => s));
```

Comment:
If we should add the title of the workitem (and it's a good thing!), I highly prefer to display one line per workitem.
Something like:

```
Workitems:
#3023 This is the title of my first workitem
#3025 This is the title of my second workitem
```